src/utils.rs

```diff
@@ -8,57 +10,89 @@ use serenity::model::{
 use std::collections::HashMap;
 
 pub fn get_num_guilds(guilds: &Vec<GuildStatus>) -> String {
-    if guilds.len() == 1 {
-        String::from("1 guild")
-    } else {
-        format!("{:?} guilds", guilds.len())
-    }
+    format!(
+        "{} guild{}",
+        guilds.len(),
+        if guilds.len() > 1 { "" } else { "s" }
+    )
```

Comment:
I slightly prefer this style because the structure of the format is only defined once. But either is fine really

Reply:
Ah so that's how you basically do ternary statements, with inline `if`s. Good to know